genie-web/src/test/groovy/com/netflix/genie/web/controllers/DtoConvertersSpec.groovy

```diff
@@ -23,20 +23,7 @@ import com.netflix.genie.common.dto.ApplicationStatus
 import com.netflix.genie.common.dto.ClusterCriteria
 import com.netflix.genie.common.dto.ClusterStatus
 import com.netflix.genie.common.dto.CommandStatus
-import com.netflix.genie.common.internal.dto.v4.Application
-import com.netflix.genie.common.internal.dto.v4.ApplicationMetadata
-import com.netflix.genie.common.internal.dto.v4.ApplicationRequest
-import com.netflix.genie.common.internal.dto.v4.Cluster
-import com.netflix.genie.common.internal.dto.v4.ClusterMetadata
-import com.netflix.genie.common.internal.dto.v4.ClusterRequest
-import com.netflix.genie.common.internal.dto.v4.Command
-import com.netflix.genie.common.internal.dto.v4.CommandMetadata
-import com.netflix.genie.common.internal.dto.v4.CommandRequest
-import com.netflix.genie.common.internal.dto.v4.Criterion
-import com.netflix.genie.common.internal.dto.v4.ExecutionEnvironment
-import com.netflix.genie.common.internal.dto.v4.ExecutionResourceCriteria
-import com.netflix.genie.common.internal.dto.v4.JobMetadata
-import com.netflix.genie.common.internal.dto.v4.JobRequest
+import com.netflix.genie.common.internal.dto.v4.*
```

Comment:
IDEA defaults to `*` if you have more than 10.
Ok to let it do this, or shall we always disallow wildcard imports?
(we can even edit gradle configuration for idea so it generates a project with this value set to `never`)

Reply:
Yeah I don't like the wildcard imports. I actually disabled it for java but I guess I forgot for Groovy. I've changed it in my IDE now. We can change it for the IDEA gradle plugin if you want I'm fine with that I just don't use it.